scorebox/standard/the_killing_floor/map.xml

```diff
@@ -0,0 +1,355 @@
+<map proto="1.4.2">
+<name>The Killing Floor</name>
+<version>1.0.0</version>
+<phase>development</phase>
+<objective>Pillar up to the Scorebox using TNT</objective>
+<created>2023-06-01</created>
+<authors>
+    <author uuid="c25a6f7b-4c42-40da-8cd6-add53f0c84eb"/> <!-- arcadeboss -->
+</authors>
+<gamemode>scorebox</gamemode>
+<teams>
+    <team id="black-team" color="black" max="20">Black Hearts</team>
```

Comment:
I would recommend avoiding using pitch black for team color as it makes it difficult to read the team name and its member in the chat, especially given that this map takes place at night. If you would like to stick to darker colors, I would recommend `dark gray` (color code 8) or `gray` (color code 7).
```suggestion
    <team id="black-team" color="dark gray" max="20">Black Hearts</team>
```

Reply:
Hi Pear, thank you so much for looking at my game, appreciate your time and thoughtfulness.   
And, you are exactly right, there is no argument: there are issues with Black.

But I am trapped in a prison of design choices made by Minecraft and myself.   It would be embarrassing to say how many times I changed the team color from Black to Gray to Dark Gray and back again, testing and anguishing over my options, cursing Mojang for not adjusting the shadow layer text of black from also being black.  (What were they thinking?)

In the end I made a final decision, I could not stand the look of even Dark Gray for the Black Hearts, there was only one choice, there was only ever one choice.  Damn the consequences, the Black Hearts must be Black.

(Going to avoid these team colors in the future, due to these issues.   If players actually have interest in playing the game again and highlight this issue, will compromise on the color.)

Reply:
@TheRealPear - Based on your suggestion,  I updated the XML on my server and did more testing.

I tried the Dark Gray again, but it actually has more readability issues than Black so that isn't going to be a suitable option when considering user concerns.   

I also tried the Gray, and it is legible, but it appears almost identical to White, which would be weird and confusing.

There are a few moments when the Black text is over Black background in Spawn, but it seems in practice that the Arena area, where gameplay mainly takes place, contains so much contrast that legibility isn't a problem.